test/simplehashregistrar_test.js

```diff
@@ -88,9 +88,12 @@ describe('SimpleHashRegistrar', function() {
 
 	it('starts auctions', function(done) {
 		async.series([
-			function(done) {
+			function(done) { 
+				advanceTime(days(100), done); 
```

Comment:
Can you export the 'launchlength' constant and use it here and elsewhere instead of 100 days?